### PR TITLE
8 custom line

### PR DESCRIPTION
**Pull request by wizzy25**
#### Reviewers

- [ ] Reviewer 1 has approved the PR
- [ ] Reviewer 2 has approved the PR

#### TODO

- Tests
    - [ ] Unit
    - [ ] Integration
    - [ ] Acceptance
- [ ] Documentation

### CODE REVIEW

File: src/orders-update.js [hunk 1]
```diff
@@ -64,6 +64,21 @@ export default class OrdersUpdate {
       })
   }
 
+  resolveState (items) {
```
Comment (hisabimbola) [inline src/orders-update.js]: We should be consistent by adding comments to this method like every other method

Cc @Siilwyn

**Review by hisabimbola — APPROVED** [review]

File: src/orders-update.js (hunk 1 above)
Comment (Siilwyn) [reply src/orders-update.js]: // Replace state references
// resolveState :: Array -> Promise -> Object

File: tests/integration/orders-update.spec.js [hunk 2]
```diff
@@ -49,12 +49,16 @@ test('the module should modify an existing order', (t) => {
     .then(() => Promise.all([
       ordersUpdate.client.states.create(stateSamples[0]),
       ordersUpdate.client.states.create(stateSamples[1]),
+      ordersUpdate.client.states.create(stateSamples[2]),
```
Comment (Siilwyn) [inline tests/integration/orders-update.spec.js]: Let's reuse the existing states (less API calls needed = faster tests).

File: tests/unit/orders-update.spec.js [hunk 3]
```diff
@@ -421,20 +427,28 @@ test(`expandReferences
     .withArgs(mockOrder.syncInfo[0].channel, 'channel', 'channels')
     .returns(channelResult)
 
-  const stateResult = {
+  const lineStateResult = {
     typeId: 'state',
     id: '53 65 6c 77 79 6e',
   }
+  const customLineStateResult = {
```
Comment (Siilwyn) [inline tests/unit/orders-update.spec.js]: Could also just use 'lineStateResult' since the stubbed function is not part of the test.

**Review by Siilwyn — APPROVED** [review]

**Pull request merged**